index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Portfolio</title>
+        <script src="https://kit.fontawesome.com/8a30dfc1c1.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="style.css">
+
+    </head>
+
+      <div class="headline-wrapper">
```

Comment:
- [ ] To make the html more semantic, please replace `<div>` tag to `<section>` tag, as this is supposed to be a section